icu4c/source/i18n/measunit.cpp

```diff
@@ -55,44 +54,15 @@ static const int32_t gOffsets[] = {
     404,
     408,
     423,
-    426,
-    432,
-    442,
-    446,
+    424,
+    430,
+    440,
+    444,
+    448,
     450,
-    452,
-    486
```

Comment:
Renumbering due to removal of duplicate "percent" and "permille" under the "none" category.